libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/InspectionView/InspectionView.tsx

```diff
@@ -96,6 +96,7 @@ export class InspectionView extends React.PureComponent<
       sortArray: [],
       sortingSeriesIndex: undefined
     };
+    this.props.selectedCohort.cohort.sort();
```

Comment:
this may have side effect
if props used in sibling components

Reply:
ok this may need to be re-examined in the other components